package.json

```diff
@@ -63,6 +64,14 @@
   "peerDependencies": {
     "knex": ">=0.6.10 <0.13.0"
   },
+  "buildDependencies": [
+    "babel-cli",
+    "babel-runtime",
```

Comment:
`babel-runtime` shouldn't actually be needed here since it's already in `dependencies`, right?